src/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -1,5 +1,7 @@
-//  Copyright (c) 2021 FlyByWire Simulations
-//  SPDX-License-Identifier: GPL-3.0
+// Copyright (c) 2021-2022 FlyByWire Simulations
+// Copyright (c) 2021-2022 Synaptic Simulations
```

Comment:
```suggestion
```

Reply:
This is in the FMGC, therefore intended.